Plugins/NativeEngine/Source/NativeEngine.cpp

```diff
@@ -1137,9 +1137,8 @@ namespace Babylon
 
         texture->Width = width;
         texture->Height = height;
-        texture->CreationFlags = BGFX_TEXTURE_NONE;
-        texture->Flags = BGFX_TEXTURE_NONE | BGFX_SAMPLER_NONE | BGFX_CAPS_TEXTURE_2D_ARRAY;
-        texture->Handle = bgfx::createTexture2D(width, height, generateMips, depth, Cast(format), texture->Flags);
+        uint64_t flags{BGFX_TEXTURE_NONE | BGFX_SAMPLER_NONE | BGFX_CAPS_TEXTURE_2D_ARRAY};
```

Comment:
I can't leave a comment for code not being changed, but you need to throw Napi::Error instead of std::runtime_error above.

Reply:
Replaced std::exception with Napi::Error in the latest commit.